docs/source/session.rst

```diff
@@ -66,6 +66,26 @@ declared as part of the job.
     df = spark.read_ext.by_url('cassandra://<cassandra-host>/<db>/<table>?consistency=QUORUM')
 
 
+Custom Maven repositories
+-------------------------
+
+**Why**: If you have private maven repository, this is the way how to point spark to it to make package lookup.
+
+**For example**: Let's assume your maven repository is available on: http://my.repo.net/maven,
```

Comment:
```
 --packages                  Comma-separated list of maven coordinates of jars to include
                              on the driver and executor classpaths. Will search the local
                              maven repo, then maven central and any additional remote
                              repositories given by --repositories. The format for the
                              coordinates should be groupId:artifactId:version.
  --exclude-packages          Comma-separated list of groupId:artifactId, to exclude while
                              resolving the dependencies provided in --packages to avoid
                              dependency conflicts.
  --repositories              Comma-separated list of additional remote repositories to
                              search for the maven coordinates given with --packages.
```
Trying to understand the order:
- local maven (local fs)
- maven central
- packages
Correct?